parmed/openmm/parameters.py

```diff
@@ -159,9 +165,8 @@ def _remediate_residue_template(cls, params, residue):
                     # Remove nonphysical H-H bonds
                     LOGGER.debug('Deleting H-H bond from water residue {}'.format(residue.name))
                     residue.delete_bond(bond)
-                elif (bond.atom1.atomic_number == 0) or (bond.atom2.atomic_number == 0):
-                    LOGGER.debug('Deleting bonds to virtual sites in residue {}'.format(residue.name))
-                    residue.delete_bond(bond)
+                else:
+                    LOGGER.debug('keeping %s to %s %s' %(str(bond.atom1), str(bond.atom2), bond.atom2.element_name))
```

Comment:
Minor nit, but you don't need the `str()` around the atoms, and you should replace the `%` operator with a comma:

```python
LOGGER.debug('keeping %s to %s %s', bond.atom1, bond.atom2, bond.atom2.element_name)
```

Reply:
Fixed!